### ISSUE TITLE
Failed to fetch

### ISSUE THREAD

**Issue by coding-sunshine** [VemtoOrg/vemto2-issues#140]
**Error Message:**
Failed to fetch

**Stack:**
```
TypeError: Failed to fetch
    at D5.checkLicense (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.ffee13a7.js:122:9102)
    at file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.ffee13a7.js:135:3939
    at file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.ffee13a7.js:16:21770
    at Xv (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.ffee13a7.js:16:2031)
    at Ag (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.ffee13a7.js:16:2102)
    at OV.e.__weh.e.__weh (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.ffee13a7.js:16:21651)
    at d7 (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.ffee13a7.js:16:3508)
    at Jpe (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.ffee13a7.js:16:3818)
```

---

Vemto 2.0.14
darwin 23.2.0
Locale: en-GB
App version: 2.0.14

**Comment by TiagoSilvaPereira** [comment]
@coding-sunshine seems it was just a network error when trying to fetch the license. Is this still happening?

**Comment by TiagoSilvaPereira** [comment]
Fixed on Vemto 2.0.15
